sbin/common-functions.sh

```diff
@@ -21,8 +21,8 @@ FREETYPE_FONT_VERSION=${FREETYPE_FONT_VERSION:-2.4.0}
 
 determineBuildProperties() {
 
-    OS_KERNEL_NAME=$(uname | awk '{print tolower($0)}')
-    OS_MACHINE_NAME=$(uname -m)
+    export OS_KERNEL_NAME=$(uname | awk '{print tolower($0)}')
+    export OS_MACHINE_NAME=$(uname -m)
```

Comment:
I think that both of these vars should be globally readable throughout the build process